app/src/main/res/values/strings.xml

```diff
@@ -20,4 +20,20 @@
     <string name="status_message_default">Brought to you live, by aTox</string>
     <string name="tox_id">Tox ID</string>
     <string name="tox_id_error_length">ToxID must be 76 characters</string>
+    <string name="title_activity_settings">Settings</string>
+
+    <!-- Preference Titles -->
+    <string name="tox_header">Tox</string>
+    <string name="user_header">User</string>
+
+    <!-- Messages Preferences -->
+    <string name="username_title">Username</string>
+    <string name="status_title">Status Message</string>
+    <string name="state_title">State</string>
+
+    <!-- Tox Preferences -->
+    <string name="proxy_title">Proxy</string>
+    <string name="proxy_address_title">Proxy Address</string>
+    <string name="proxy_address_hint">Your proxy here</string>
```

Comment:
I think you forgot to use this?

Reply:
app:hint is not a an attribute and app:summary didn't work so I'm just going to delete it